apps/react-app/src/pods/manfred-json-input/manfred-json-input.component.tsx

```diff
@@ -1,7 +1,13 @@
 import React from 'react';
 import {useState} from 'react';
 
-export const ManfredJsonInput: React.FC = () => {
+interface Props {
+  handleData: (text: string) => void;
```

Comment:
When there is a prop a callback / event should be alway "On....", and when you write the implementation it should be "Handle..."

We should define a name more meaninful in this context, for instance

````ts
OnSetManfredJson
```

Another option

```ts
onSubmitManfredJson
```